pkg/controller/provider/web/ocp/client.go

```diff
@@ -69,3 +69,15 @@ func (r *Resolver) Path(object interface{}, id string) (path string, err error)
 
 	return
 }
+
+//
+// Resource finder.
+type Finder struct {
+	base.Client
+}
+
+//
+// Find a resource by ref.
+func (r *Finder) ByRef(client base.Client, resource interface{}, ref base.Ref) (err error) {
```

Comment:
Added just to comply with the interface.  I don't anticipate this will ever be implemented.